build/int.cloudbuild.yaml

```diff
@@ -78,4 +78,4 @@ tags:
 - 'integration'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.12.2'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.13.0'
```

Comment:
nit: pin to `0.13`
Also update tag here: https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/Makefile#L21